mapdata/WarcraftLegacies/AbilityData/Skin/1513238849.json

```diff
@@ -40,7 +40,7 @@
       "Level": 1,
       "Id": 828536161,
       "Type": 3,
-      "Value": "Resurrect a nearby fallen ally to fight again every 5 seconds."
+      "Value": "Resurrect a nearby fallen ally to fight again every \u003CA12Z,Cool1\u003E seconds."
```

Comment:
Interesting, does this actually display correctly in game? I thought this ability was triggered?

Reply:
Im pretty sure it does cause the banner that gets spawned  has an auto cast ability  with a tooltip so i assumed that was the tooltip and it displays 5 seconds there unless its something else completly

Reply:
Ah okay, sweet!